outgoing_routing/models/stock_picking_mixin.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 VentorTech OU
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+
+from odoo import models
+
+import logging
+
+_logger = logging.getLogger(__file__)
+
+
+class StockPickingMixin(models.AbstractModel):
+    _name = 'stock.picking.mixin'
+    _description = 'Stock Picking Mixin'
+
+    def _read_record(self, record_tuple):
+        """
+        record_tuple = (
+            ('id', 100),
+            ('_type', 'stock.move.line'),
+        )
+
+        id:: number (int)
+        _type:: 'stock.move.line' or 'stock.package_level' (str)
+        """
+        record_dict = dict(record_tuple)
+        record = self.env[record_dict['_type']].browse(record_dict['id'])
+        record_dict.update(record.read()[0])
+        return record_dict
+
+    def serialize_record_merp(self, rec_id):
+        """Record serialization for the mERP app."""
```

Comment:
It is not mERP app, it is Ventor App

Also rename metod serialize_record_merp-> serialize_record_ventor

Reply:
sure